render/detailed_node.go

```diff
@@ -128,22 +123,29 @@ func OriginTable(r report.Report, originID string) (Table, bool) {
 	return Table{}, false
 }
 
-func endpointOriginTable(nmd report.NodeMetadata) (Table, bool) {
+func connectionDetailsRows(endpointTopology report.Topology, originID string, nmd report.NodeMetadata) []Row {
 	rows := []Row{}
-	for _, tuple := range []struct{ key, human string }{
-		{"addr", "Endpoint"},
-		{"port", "Port"},
-	} {
-		if val, ok := nmd[tuple.key]; ok {
-			rows = append(rows, Row{Key: tuple.human, ValueMajor: val, ValueMinor: ""})
+	local := fmt.Sprintf("%s:%s", nmd["addr"], nmd["port"])
+	adjacencies := endpointTopology.Adjacency[report.MakeAdjacencyID(originID)]
+	sort.Strings(adjacencies)
+	for _, adj := range adjacencies {
+		if _, address, port, ok := report.ParseEndpointNodeID(adj); ok {
+			rows = append(rows, Row{
+				Key:        local,
+				ValueMajor: fmt.Sprintf("%s:%s", address, port),
+			})
 		}
 	}
+	return rows
+}
+
+func connectionDetailsTable(connectionRows []Row) Table {
 	return Table{
-		Title:   "Origin Endpoint",
+		Title:   "Connection Details",
 		Numeric: false,
-		Rows:    rows,
+		Rows:    append([]Row{{Key: "Local", ValueMajor: "Remote"}}, connectionRows...),
```

Comment:
Oof. I guess this will do for now, but let's file an issue to modify the Table data type to more properly support this kind of thing. (Maybe we can have it be an interface with a RenderToHTML method, something like this.)